PIL/Image.py

```diff
@@ -1236,6 +1307,7 @@ def getpalette(self):
             return None  # no palette
 
     def getpixel(self, xy):
+        # type: (Coord) -> Tuple[Any]
```

Comment:
I think `Union[Color, Tuple[Color, ...]]`looks right. But what do you mean by alias `Color = [something]`? As in `Color` refers to a list of some values?